torch/csrc/distributed/c10d/ProcessGroupUCC.hpp

```diff
@@ -165,6 +165,12 @@ class TORCH_API ProcessGroupUCC : public ProcessGroup {
         const char* prof_title,
         const c10::intrusive_ptr<ProcessGroupUCCLogger>& logger)
         : ProcessGroup::Work(-1, opType, prof_title), logger_(logger) {}
+    WorkUCC(
```

Comment:
Do we still need the constructor that doesn't have inputs?

Reply:
There's one use left of the old ctor but it looks like it should use the new one.

Reply:
Removed old constructor. Couldn't find the one use left of the old constructor that @kumpera mentioned.